tests/clients/test_cached_backend_client.py

```diff
@@ -27,3 +27,20 @@ def test_init():
         )
         assert client.initialized_event.wait(timeout=0.1)
         client.close()
+
+
+def test_should_update_store():
+    with mock.patch("lekko_client.clients.cached_backend_client.Event") as mock_event:
+        mock_event().is_set.return_value = True
+        with mock.patch("lekko_client.clients.distribution_client.DistributionServiceStub") as mock_stub:
```

Comment:
nit: you can use `with A() as a, B() as b:` syntax to reduce nesting

Reply:
Nice, I completely forgot about that